comment_spell_check.py

```diff
@@ -377,6 +387,11 @@ def add_dict(enchant_dict, filename, verbose=False):
 
     # You better not have more than 1 word in a line
     for wrd in lines:
+        if not word_check(wrd):
```

Comment:
Change line to:
```
if not word.replace("'","").isidentifier():
```

Reply:
Yeah, your way is more clever than mine.  My brain just doesn't work that way.  I'll update it.